### PR TITLE
Handles Integrity Error for multiple mobilisation visits created

### PR DESCRIPTION
**Pull request by WinnyTroy**

### CODE REVIEW

File: mspray/apps/main/views/mobilisation.py [hunk 1]
```diff
@@ -1,19 +1,24 @@
...
     def post(self, request):
         """Handle a Mobilisation submission."""
-        create_mobilisation_visit(request.data)
-        return Response(status=status.HTTP_201_CREATED)
+        try:
+            create_mobilisation_visit(request.data)
+            return Response(status=status.HTTP_201_CREATED)
+        except IntegrityError:
+            logger.exception("Mobilisation submission already exists.")
+            return Response(status=status.HTTP_202_CREATED)
```
Comment (ukanga) [inline mspray/apps/main/views/mobilisation.py]: Could we re-write the `try` block like to be more clear? e.g
```python
        try:
            create_mobilisation_visit(request.data)
        except IntegrityError:
            logger.exception("Mobilisation submission already exists.")
            status = status.HTTP_202_CREATED
        else:
            status = status.HTTP_201_CREATED

        return Response(status=status)

Comment (ukanga) [inline mspray/apps/main/views/mobilisation.py]: Could you add a test that checks we return the `status_code` `200` when we attempt to create a duplicate mobilisation record?
